### ISSUE TITLE
Quit button has no texture in case-sensitive OSes

### ISSUE THREAD

**Issue by Exaxxion** [Drathonix/Nuclear-Earth-Reirradiated#3]
Pack release version 2.1.2

**Description:**
The image resource filename for the Quit button is capitalized, leading to a failure to load the texture on operating systems that respect case sensitivity (basically, everything that isn't Windows).

![image](https://user-images.githubusercontent.com/11907282/117728046-bfc0d680-b19d-11eb-8646-9fd944628b6a.png)

**Solution:**
If you rename the file `overrides/resources/mainmenu/Exit.png` to lowercase it should resolve this problem. Resources by convention are all lowercase file names in 1.11+ / pack format 3.

**Comment by Drathonix** [comment]
I'll be changing the file name next update. 
However, I don't see how this could be an OS issue. I know the menu works on OSes other than windows, it works on mac from what I've seen so far. Obviously, the only possibility is you're running something else or a different version.

I'd love to know what OS you're running as it'd give me more information on this concept, I've been under the impression that the JVM minecraft runs on works similarly on most OSes and wouldn't have trouble reading lowercase resources, which it doesn't on windows... So is windows CaseSensitive then? The casesensitivity issue is possibly a result of JVM differences on certain machines in that case. What operating system doesn't support reading capitalized characters?

Also you're running Version 2.1.2, update to 2.1.3a
Regardless though, the problem is in both versions

**Comment by Exaxxion** [comment]
I'm running Linux. Windows is case-insensitive, meaning that it considers "Exit.png" and "exit.png" the same file. Perhaps Mac is the same, but as a distant derivative of Unix I figured it would have kept that convention.

Linux treats them as different files: if it's defined as "exit.png" somewhere and the file is actually "Exit.png" you get a missing texture placeholder.

We had the same problem with a texture a while back: https://github.com/OmnifactoryDevs/Omnifactory/issues/168.

I grabbed the latest release version off CurseForge a few days ago, perhaps it actually is 2.1.3a (I don't have the zip, I installed it directly through MultiMC) and I was just going by what the Window title bar says.

Edit: comparing the versions of mods to the release notes of 2.1.3a, I'm pretty sure I am on it.
